sphinx_gallery/py_source_parser.py

```diff
@@ -27,12 +34,16 @@ def get_docstring_and_rest(filename):
     # can't use codecs.open(filename, 'r', 'utf-8') here b/c ast doesn't
     # seem to work with unicode strings in Python2.7
     # "SyntaxError: encoding declaration in Unicode string"
-    with open(filename, 'rb') as f:
-        content = f.read()
+    with open(filename, 'rb') as fid:
```

Comment:
`fid` is a name I constantly encounter, so I just took it as convention. The reason to change is that my linter is always complaining about single letter variables